tests/tts_tests/test_vits_speaker_emb_train_upsampling_vocoder_approach.py

```diff
@@ -0,0 +1,90 @@
+import glob
```

Comment:
We don't need this one too if you tests things in ```test_vits.py```

Reply:
Yeah, Done